messaging-experiments.json

```diff
@@ -343,7 +343,13 @@
             },
             "targeting": "recentVisits|length > 1",
             "frequency": {
-              "lifetime": 3
+              "lifetime": 3,
+              "custom": [
```

Comment:
Experiment CFRs did not have a daily limit to spread out how often they are seen.